lib/kb/models/hubspot.rb

```diff
@@ -0,0 +1,34 @@
+module KB
+  class Hubspot < BaseModel
+    include Queryable
+
+    kb_api :hubspot
+
+    def self.attributes_from_response(response)
+      response.transform_keys(&:underscore).transform_keys(&:to_sym).slice(*FIELDS)
+    end
+
+    private_class_method :attributes_from_response
+
+    STRING_FIELDS = %i[kb_key hubspot_id].freeze
+    DATE_FIELDS = %i[last_updated_at].freeze
+    FIELDS = [*STRING_FIELDS, *DATE_FIELDS].freeze
+
+    define_attribute_methods(*FIELDS)
+
+    STRING_FIELDS.each do |field|
+      attribute field, :string
+    end
+
+    DATE_FIELDS.each do |field|
+      attribute field, :date
+    end
+
+    def self.relationship(model, key)
```

Comment:
Renaming the `key` attribute might help on clarity:
```suggestion
    def self.relationship(model, model_key)
```

Then the method itself returns an instance of a HubspotRelationship so what do you think of simply naming it: 
```suggestion
    def self.find(model, model_key)
```